semgrep-core/src/runner/Run_semgrep.ml

```diff
@@ -546,14 +603,54 @@ let semgrep_with_raw_results_and_exn_handler config =
    * semgrep-core -rules <(curl https://semgrep.dev/c/p/ocaml) ...
    *)
   let rules_file = replace_named_pipe_by_regular_file rules_file in
+  let extractors_file =
+    if config.extractors_file <> "" then
+      Some (replace_named_pipe_by_regular_file config.extractors_file)
```

Comment:
I doubt we want to define extractor rule via named pipe. 
I think this replace_named_pipe_... was added so you could do in the shell semgrep  -e foo  $(<echo foo>) or something like that, but for -pp I don't think we need that feature.

Reply:
Hmm actually wouldn't the same logic as in the comment apply, like if you wanted to run with `-pp <(curl $URL)`?

Reply:
Oh you re right. Might be good to add (* this allows -pp (curl ...)